cmd/av/pr_status.go

```diff
@@ -150,6 +159,28 @@ var prStatusCmd = &cobra.Command{
 			)
 		}
 
+		// Get Associated BotPR info
+		botPullRequest := query.GithubRepository.PullRequest.BotPullRequest
+		if botPullRequest.Number == 0 {
+			// no associated botPR
+			return nil
+		}
+
+		_, _ = fmt.Fprint(os.Stderr, "Associated PR #", botPullRequest.Number, " Required Checks\n")
```

Comment:
Can we call it bot pull request? Let's try to use consistent language. It's called "associated PR" in a few places, but we should either change that to "bot pull request" or change what we call it in the API so those two things match.